R/install.R

```diff
@@ -10,13 +10,13 @@
 #' @param remove_existing_env delete the conda environment if already exists
 #' @return None
 #' @export
-install_azureml <- function(version = "1.5.0",
-                            envname = NULL,
+install_azureml <- function(version = "1.8.0",
```

Comment:
do we need to perform some tests before bumping the python sdk version?

Reply:
what additional tests do you suggest? what have you guys done for the previous bumps? I've tested with our vignettes as well

Reply:
I generally run some e2e tests to test the validation but I think testing vignettes is also good enough